src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -22,8 +23,8 @@ function actionFactory(e, ctx) {
     if (e.nativeEvent.detail == 2) {
       return removeNoteAction(ctx, note)
     } else {
-      const local = pointSub(cpos, note)
-      const type = getDragPositionType(local.x, note.width)
+      const offset = pointSub(local, note)
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->